tests/csm/rest/test_iam_users.py

```diff
@@ -968,3 +971,450 @@ def test_37020(self):
+        self.created_iam_users.add(uid)
+        get_resp = self.csm_obj.get_iam_user(user=uid)
+        assert_utils.assert_true(get_resp.status_code == HTTPStatus.OK, "Get IAM user failed")
+        access_key_init = get_resp.json()["keys"][0]['access_key']
+        valid_key = self.csm_conf["test_36448"]["valid_key"]
+        self.log.info("Adding key to user")
+        add_resp = self.csm_obj.add_key_to_iam_user(uid=uid, access_key=access_key_init,
+                                                    secret_key=valid_key)
+        assert_utils.assert_true(add_resp.status_code == HTTPStatus.OK, "Add key failed")
+        assert_utils.assert_true(len(add_resp.json()) == 1, "More than 1 keys are received")
+        access_key = add_resp.json()[0]['access_key']
+        secret_key = add_resp.json()[0]['secret_key']
+        assert_utils.assert_true(access_key == access_key_init, "Access key is not matching")
+        assert_utils.assert_true(secret_key == valid_key, "Secret key is not matching")
+        bucket_name = "iam_user_bucket_" + str(int(time.time()))
+        self.log.info("Create bucket and perform IO")
+        s3_obj = S3TestLib(access_key=access_key,
+                           secret_key=secret_key)
+        bucket_name = bucket_name.replace("_", "-")
+        status, resp = s3_obj.create_bucket(bucket_name)
+        assert_utils.assert_true(status, resp)
+        test_file = "test-object.txt"
+        file_path_upload = os.path.join(TEST_DATA_FOLDER, test_file)
+        if os.path.exists(file_path_upload):
+            os.remove(file_path_upload)
+        if not os.path.isdir(TEST_DATA_FOLDER):
+            self.log.debug("File path not exists, create a directory")
+            system_utils.execute_cmd(cmd=common_cmd.CMD_MKDIR.format(TEST_DATA_FOLDER))
+        system_utils.create_file(file_path_upload, self.file_size)
+        self.log.info("Step: Verify put object.")
+        resp = s3_obj.put_object(bucket_name=bucket_name, object_name=test_file,
+                                 file_path=file_path_upload)
+        self.log.info("Removing uploaded object from a local path.")
+        os.remove(file_path_upload)
+        assert_utils.assert_true(resp[0], resp[1])
+        self.log.info("Step: Verify get object.")
+        resp = s3_obj.get_object(bucket_name, test_file)
+        assert_utils.assert_true(resp[0], resp)
+        self.log.info("##### Test completed -  %s #####", test_case_name)
+
+    @pytest.mark.csmrest
+    @pytest.mark.lc
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-37777')
+    def test_37777(self):
+        """
+        Create key request with existing access key of another user
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        self.log.info("[START] Creating IAM user")
+        uids = []
+        access_keys = []
+        for user in range(2):
```

Comment:
can we accept 2 from the test config?

Reply:
This is fixed, not going to change. So, will keep as it is.